firebase_admin/_token_gen.py

```diff
@@ -291,14 +311,16 @@ def verify(self, token, request):
                 '{0} expects {1}, but was given a custom '
                 'token.'.format(self.operation, self.articled_short_name))
         elif not header.get('kid'):
+            if _auth_utils.is_emulated():
```

Comment:
Use a local variable please. Do not call `is_emulated()` more than once per verification.